todolist/settings.py

```diff
@@ -62,12 +62,12 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': 'mysql',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+        'ENGINE': os.getenv("DB_ENGINE", "mysql.connector.django"),  
+        'NAME': os.getenv("DB_NAME", "app_db"),
+        'USER': os.getenv("DB_USER", "app_user"),
+        'PASSWORD': os.getenv("DB_PASSWORD", "1234"),
```

Comment:
While using environment variables for database credentials is a good practice, ensure that these environment variables are set securely and not exposed in any public or insecure manner.